centreon/www/include/configuration/configObject/contact/DB-Func.php

```diff
@@ -197,6 +197,49 @@ function disableContactInDB($contact_id = null, $contact_arr = array())
     }
 }
 
+/**
+ * Unblock contacts in the database
+ *
+ * @param int|array|null $contact Contact ID, array of contact IDs or null to unblock all contacts
+ */
+function unblockContactInDB($contact = null)
+{
+    global $pearDB, $centreon;
+
+    if (is_null($contact)) {
+        return;
+    }
+
+    if (!is_array($contact)) {
+        $contact = [$contact => "1"];
+    }
+
+
+    $bindContactIds = [];
+    foreach (array_keys($contact) as $contactId) {
+        $bindContactIds[':contact_' . $contactId] = $contactId;
+    }
+//  implode ids for  IN() clause
+    $idPlaceholders = implode(', ', array_keys($bindContactIds));
+// retrieve the users and add log
+    $updateQuery = "UPDATE contact SET blocking_time = null WHERE contact_id IN ($idPlaceholders)";
+    $statement = $pearDB->prepare($updateQuery);
```

Comment:
Rename `$statement` in `$updateStatement` to be consistent with `$selectStatement` in the naming.